docs/transaction-building.md

```diff
@@ -0,0 +1,264 @@
+```ts
+new TransactionBuilder(creationHeight)
+  .from(inputs)
+  .to(
+    new OutputBuilder("1000000", "9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j")
+      .mintToken({ // [!code focus]
+        amount: "100", // the amount of tokens being minted without decimals // [!code focus]
+        name: "TestToken", // the name of the token // [!code focus]
+        decimals: 2, // the number of decimals  // [!code focus]
+        description: "This is a test token minted with Fleet SDK" // [!code focus]
+      }) // [!code focus]
+  );
+```
+
+:::warning
+Only one token can be minted per transaction.
+:::
+
+## Step. 5: Set the change address
+
+Often, all the funds included in the input boxes are not used by the transaction. An address needs to be set to receive these funds. This is referred to as the change from the eUTxO transaction.
+
+To do so, you must call the `sendChangeTo()` method and pass an address as the only parameter.
+
+In the following example, we are sending the change to the `9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j` address.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .sendChangeTo("9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j"); // [!code focus]
+```
+
+## Step. 6: Set the fee amount
+
+Each transaction on Ergo needs to have a mining fee box, you can use the `payFee()` method to specify the fee amount in `nanoergs`.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .payFee("1100000"); // [!code focus]
+```
+
+Alternatively, you can use the `payMinFee()` method to add the min recommended miner fee amount, which currently is `1100000 nanoergs` or `0.0011 ERG`.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .payMinFee(); // [!code focus]
+```
+
+## Step. 7: Build
+
+Now that all pieces are put together, it's time to build the transaction and obtain the **Unsigned Transaction** object. For that you can use the `build()` method.
+
+The `build()` method must be the last called method on the `TransactionBuilder` as it will do all necessary validations, input selection, change calculations and return an unsigned transaction object which can be signed by any Ergo signing tool.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+   // ...
+  .build(); // [!code focus]
+```
+
+By default, the `build()` method will return a default unsigned transaction object, if you want to send it to be signed by the user's wallet through [dApp Connector protocol](./wallet-interaction.md), you should include "EIP-12" as a parameter in the `build()` method so it will include additional information required by the protocol.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+   // ...
+  .build("EIP-12"); // [!code focus]
+```
+
+## Example
+
+Building and signing a transaction using the dApp Connector protocol as a source of blockchain data.
```

Comment:
```suggestion
Build and sign a transaction using the dApp Connector protocol to access blockchain data.
```

Reply:
I guess "to interact" fit better here, will rephrase this to be a bit more precise.